admin/sql/CreateTables.sql

```diff
@@ -21,6 +21,7 @@ CREATE TABLE application
 
 CREATE TABLE area_type ( -- replicate
     id                  SERIAL, -- PK
+    gid                 uuid NOT NULL,
```

Comment:
Yes, unfortunately Postgres (even in 9.5) still doesn’t allow freely ordering columns. :-/